src/core/src/bootstrap/ConfigurationFactory.py

```diff
@@ -278,7 +278,6 @@ def get_vm_cloud_type(self):
         for i in range(0, Constants.MAX_IMDS_CONNECTION_RETRY_COUNT):
             try:
                 res = urlreq.urlopen(request, timeout=2)
-                print(res.get_code())
```

Comment:
why remove print statement instead of correcting the function name? It's always good to have logs on API response. Will help debug when needed on a machine